python/cudf/cudf/_libxx/cpp/nvtx.pxd

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+
+from libc.stdint cimport uint32_t
```

Comment:
We're following the cpp header layout for Cython headers: move file to `_libxx/cpp/utilities/nvtx_utils.pxd`